question_queue/models.py

```diff
@@ -35,6 +35,11 @@ class Question(models.Model):
     last_updated = models.DateTimeField(auto_now=True)
     in_person = models.BooleanField(default=False)
     asked_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    hidden = models.BooleanField(default=False)
+
+
+class QueueQuestion(Question):
+    answered_by = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
I'm a little confused about how this is different than adding the "answered_by" field to the Question model and just defaulting it to empty.

Reply:
In the context of a forum the question is implicitly answered by a question where as with the queue the question is explicitly answered by a coach.

Reply:
Alright, thanks for the input. This looks good, ship it.